srcpkgs/ProTracker/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'ProTracker'
+pkgname=ProTracker
+version=0.20b
+revision=1
+hostmakedepends="subversion"
+makedepends="SDL2-devel"
+depends="SDL2"
+short_desc="Modern clone of the amiga ProTracker music tracker"
+maintainer="rc-05 <rc23@email.it>"
+license="WTFPL"
+homepage="https://16-bits.org/pt.php"
+
+do_fetch() {
+	svn checkout svn://svn.code.sf.net/p/protracker/code/trunk ${wrksrc}/pt-clone-code
+}
+
+do_build() {
+	cd ${wrksrc}/pt-clone-code
+	chmod +x make-linux.sh
+	./make-linux.sh
```

Comment:
firstly, packages should not be compiled `-march=native` set. Also it hardcodes gcc as compiler  which will break cross. There is a GNUMakefile in the same directory, it might work better for cross compiling.